templates/components/card.html

```diff
@@ -52,7 +52,7 @@
     {% endif %}
     {% if imageSrc %}
       {% if href %}
-      <a href="{{ href }}" class="tna-card__image-container" tabindex="-1">
+      <a href="{{ href }}" class="tna-card__image-container" tabindex="-1" title="{{ title }}">
```

Comment:
I added in a title here as axe complained about the lack of discernible text on the link